cmd/juno/juno.go

```diff
@@ -381,10 +391,99 @@ func GenP2PKeyPair() *cobra.Command {
 
 			pubHex := make([]byte, hex.EncodedLen(len(rawPub)))
 			hex.Encode(pubHex, rawPub)
-			fmt.Println("P2P Public Key:", string(pubHex))
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), "P2P Public Key:", string(pubHex))
+			if err != nil {
+				return err
+			}
+
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), "P2P PeerID:", id)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+}
+
+//nolint:gocyclo
+func DBSize() *cobra.Command {
+	dbSizeCmd := &cobra.Command{
+		Use:   "db-size",
+		Short: "Calculate's Juno's DB size.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			dbPath, err := cmd.Flags().GetString(dbPathF)
+			if err != nil {
+				return err
+			}
+
+			if dbPath == "" {
+				return fmt.Errorf("--%v cannot be empty", dbPathF)
+			}
+
+			pebbleDB, err := pebble.New(dbPath)
+			if err != nil {
+				return err
+			}
+
+			var totalSize, stateSizeWithoutHistory, stateSizeWithHistory uint
+
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), "Total number of DB buckets:", uint(db.SchemaIntermediateState)+1)
```

Comment:
there is a new bucket `db.Peer` maybe we should just add variable alongside bucket constants with all of them inside?

Reply:
or we can actually use this tool https://github.com/dmarkham/enumer it will generate string name for every enum value and useful methods such as `BucketValues()[]Bucket` which can be used in methods below like `if utils.AnyOf(i, db.BucketValues()...)`

Reply:
Done